example/tfvars/example.tfvars

```diff
@@ -0,0 +1,17 @@
+vm = {
+  autostart              = true
+  base_volume_image_url  = "https://cdimage.debian.org/cdimage/cloud/buster/20211011-792/debian-10-generic-amd64-20211011-792.qcow2"
+  bridge_interface_name  = "virbr0"
+  count                  = 1
+  hostname               = "example"
+  memory                 = 512
+  name_prefix            = "example"
+  network_interface_name = "ens3"
+  root_passwd            = "changeme"
```

Comment:
when passing `changeme` as value to root password the system will ask the user to change the password on first login: 

```

```

However, I am expecting the caller to use a terraform datasource provider to fetch the password value from a secrets management solution instead of keeping plain text passwords on git :)